apps/wing/src/cli.ts

```diff
@@ -150,8 +150,12 @@ async function main() {
       []
     )
     .option("-r, --rootId <rootId>", "App root id")
-    .option("-v, --value <value>", "Platform-specific value in the form KEY=VALUE", addValue, [])
-    .option("--values <file>", "Yaml file with Platform-specific values")
+    .option(
+      "--value <value>",
+      "Platform-specific values in the form KEY=VALUE or yaml file",
+      addValue,
```

Comment:
my 2 cents I think we can leave `-v` for 2 reasons.

1. `-v` would only exist on compile and test commands so users should not get an unexpected result when doing `wing -v`
2. The wing cli uses uppercase `-V` for version command